app/grandchallenge/evaluation/views/api.py

```diff
@@ -23,3 +57,96 @@ class EvaluationViewSet(ReadOnlyModelViewSet):
         *api_settings.DEFAULT_RENDERER_CLASSES,
         PaginatedCSVRenderer,
     )
+
+    @action(
+        detail=True,
+        methods=["PATCH"],
+        permission_classes=[CanClaimEvaluation],
+        serializer_class=ExternalEvaluationSerializer,
+    )
+    def claim(self, request, *args, **kwargs):
+        evaluation = self.get_object()
+
+        if not evaluation.status == Evaluation.PENDING:
+            return Response(
+                {"status": "You can only claim pending evaluations."},
+                status=400,
+            )
+
+        if request.user.claimed_evaluations.exists():
+            return Response(
+                {"status": "You can only claim one evaluation at a time."},
+                status=400,
+            )
+
+        serializer = self.get_serializer(
+            instance=evaluation, data=request.data, partial=True
+        )
+
+        if serializer.is_valid():
+            serializer.save()
+            return Response(data=serializer.data)
+        else:
+            return Response(
+                data=serializer.errors, status=HTTP_400_BAD_REQUEST
+            )
+
+    @action(
+        detail=False,
+        methods=["GET"],
+        filter_backends=[CanClaimEvaluationFilter],
+        serializer_class=ExternalEvaluationSerializer,
+    )
+    def claimable_evaluations(self, request, *args, **kwargs):
+        qs = self.filter_queryset(self.get_queryset())
+        serializer = ExternalEvaluationSerializer(
+            qs, many=True, context={"request": request}
+        )
+        return Response(serializer.data)
+
+    @action(
+        detail=True,
+        methods=["PATCH"],
+        permission_classes=[CanClaimEvaluation],
+        serializer_class=ExternalEvaluationUpdateSerializer,
+    )
+    def update_external_evaluation(self, request, *args, **kwargs):
+        evaluation = self.get_object()
+        if evaluation.status != Evaluation.CLAIMED:
+            return Response(
+                {
+                    "status": "You need to claim an evaluation before you can update it."
+                },
+                status=400,
+            )
+
+        if request.user != evaluation.claimed_by:
+            return Response(
+                {
+                    "status": "You do not have permission to update this evaluation."
+                },
+                status=403,
+            )
+
+        if (
+            evaluation.started_at - now()
+        ).seconds > settings.EXTERNAL_EVALUATION_TIMEOUT_IN_SECONDS:
+            cancel_external_evaluations_past_timeout.apply_async()
+            return Response(
+                {
+                    "status": "You can only update an evaluation within 24 hours."
```

Comment:
Hardcoded timeout, you can make the message more general in case we change it in settings (and the error message should not change with our settings in case the client uses it for matching).